build.gradle

```diff
@@ -1,14 +1,11 @@
-group = 'com.github.aphoh'
-version = '0.0.1-SNAPSHOT'
-
-buildscript {
-    repositories {
-    }
-    dependencies {
-    }
-}
-
 allprojects {
+    apply plugin: 'java'
+    group = 'com.pokegoapi'
+    version = '0.3-1'
```

Comment:
You needed the version number updated just to turn around and change it later. This changed again later in this very pull. They are version numbers, not tags. They change with a new release, not current mood.